raspberry/read_sbus/read_sbus_from_GPIO_receiver.py

```diff
@@ -51,6 +97,20 @@
                 raise
 
 
+def run_thread_every_given_interval(interval, function_to_run,  num_times_to_run=0):
+        if num_times_to_run != 1:
```

Comment:
Here `num_times_to_run` seems to be always equal to `0` so the timer is never called.

Reply:
Professor, `0 != 1` gives `True`. So, the timer gets called.

Reply:
That's true. However, what is `num_times_to_run`? Is it necessary?

Reply:
We might want some functions to run only a couple of times, like when there is a lost connection we might only want to retry a fixed number of times periodically, and if the connection is re-established we stop the thread and go back to the normal routine, if we couldn't re-establish the connection after a certain number of attempts then we take a different routine (for example landing and then shutdown routine). This will be useful in the future, but as of now it doesn't serve any purpose. Professor, let me know if I should remove it.